app/services/documents.js

```diff
@@ -0,0 +1,39 @@
+const { format, parseISO } = require('date-fns');
+const { getDocuments } = require('../api/documents')
+
+class DocumentsService {
+
+    constructor() {}
+ 
+    formatDocumentsData(documentsJSONData) {
+        return documentsJSONData.map(
+            item => {
+                const date = parseISO(item.properties.createdOn)
+                const formattedDate = format(date, 'dd/MM/yyyy')
+
+                return {
+                    ...item,
+                    properties: {
+                        ...item.properties,
+                        createdOn: formattedDate
+                    }
+                }
+            }
+        )
+    }
+
+    async getDocumentsData(id = null) {
```

Comment:
Could we separate this into two functions? One for multiple documents and one for a single document.

Just thinking we should only return an array for the multiple documents route. Unless the API bindings already deal with this?

Reply:
updated, by id is not required for this ticket. I have updated the code to reflect this.